src/openrct2/drawing/LightFX.cpp

```diff
@@ -685,20 +718,16 @@ uint32_t lightfx_get_light_polution()
     return _lightPolution_front;
 }
 
-void lightfx_add_lights_magic_vehicles()
+void lightfx_add_lights_magic_vehicles(const Vehicle* vehicle, uint16_t spriteIndex)
```

Comment:
spriteIndex is basically unused. Just remove it completely this is not required any more.

Reply:
This is no longer putting the lights on all vehicles but on just a singular vehicle change the name of the function.